tools/tracegen/TDFParser.cpp

```diff
@@ -365,15 +365,15 @@ TDFParser::getTemplate(const char *line, const char *key)
 	}
 
 	len = (pos - vpos);
-	for (int i = strlen(line); i >= 0; i--){
-		if (line[i] == ' '){
-			continue;
-		}
-		else if (line[i] == '\"'){
+	for (int i = (int)(strlen(line)) - 1; i >= 0; i--) {
```

Comment:
Sorry instead of casting the result of strlen to an int can we just give i a type of size_t? Did I miss a previous comment where someone suggested this instead?